builtin/logical/pki/storage.go

```diff
@@ -1120,5 +1120,14 @@ func (sc *storageContext) getRevocationConfig() (*crlConfig, error) {
 		return nil, err
 	}
 
+	// Automatically update existing configurations.
```

Comment:
Do we want to switch to a versioned crl config storage now that we're modifying it more frequently? But if its just these three fields, I'm fine with it.

Reply:
Yeah it will be cleaner, updated to your suggestion.